package.json

```diff
@@ -0,0 +1,46 @@
+{
+  "name": "appium.io",
+  "version": "1.0.0",
+  "description": "Documentation project for Appium.io",
+  "main": "index.js",
+  "directories": {
+    "test": "test"
+  },
+  "scripts": {
+    "serve": "bundle exec jekyll serve",
+    "build:docs": "babel-node scripts/repo.js",
+    "test": "mocha --require babel-polyfill --compilers js:babel-core/register test/",
+    "test:watch": "mocha --require babel-polyfill --watch --compilers js:babel-core/register test/"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/appium/appium.io.git"
+  },
+  "author": "Dan Graham",
+  "license": "ISC",
```

Comment:
yeah, license goes by the file in the repo, not what's named in package.json. but it's important for both to match